ui/packages/consul-ui/app/templates/dc/services/show/topology.hbs

```diff
@@ -136,7 +98,7 @@ as |nspace dc items topology|}}
     {{#if config.data}}
       <TopologyMetrics
         @nspace={{nspace}}
-        @dc={{dc.Name}}
+        @dc={{dc}}
```

Comment:
I had to pass the whole `dc` object since I need the `dc.DefaultACLPolicy` in topology-metrics/index.js

Reply:
Cool yeah good move, thanks for the explanation.